### PR TITLE
fix(issue-143): resolve periodic assertion failures 

### PR DESCRIPTION
**Pull request by pattishin**
**Issue:** #143 

Removing assertion line since we are already checking in a latter step.

### CODE REVIEW

File: infra/test/integration/testhelper.go [hunk 1]
```diff
@@ -146,9 +151,26 @@ func assertResponseContains(t *testing.T, assert *assert.Assertions, url string,
...
+		return
+	}
+
+	// Otherwise, assert that at minimum one fragment is present in responseBody
 	for _, fragment := range text {
-		assert.Containsf(responseBody, fragment, "couldn't find %q in response body", fragment)
+		var hasFragment bool = strings.Contains(responseBody, fragment)
+
+		if hasFragment {
+			assert.Truef(hasFragment, "verified %q is in response body", fragment)
```
Comment (minherz) [inline infra/test/integration/testhelper.go:168]: nit: this line will never through assert. it can be removed safely.

**Review by minherz — APPROVED** [review]
I would suggest to make a separate condition method for "oneOf" instead of using argument that a code reader has to track to the function's implementation.

File: infra/test/integration/testhelper.go (hunk 1 above)
Comment (pattishin) [reply infra/test/integration/testhelper.go:168]: This runs when `oneOf` bool flag is true and that (at minimum) one of the strings that has been passed is present in the final `responseBody`. Check out build logs https://pantheon.corp.google.com/cloud-build/builds/87736e79-f796-433f-a2f7-9de4df7daa49.

**Comment by pattishin** [comment]
> I would suggest to make a separate condition method for "oneOf" instead of using argument that a code reader has to track to the function's implementation.

That makes sense to have a cleaner implementation. This is meant to be a minimal change/ patch for the moment to unblock, but comments have been added to revisit.

**Review by muncus — APPROVED** [review]

**Comment by donmccasland** [comment]
We are in a code freeze for solutions' deployment code, but since this change only impacts the testing code, we are good to go forward on this.

**Review by donmccasland — APPROVED** [review]

**Comment by pattishin** [comment]
Appreciate it Don & Marc! ty!

**Pull request merged**